stanford/cme/certificates.yml

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ansible-playbook
```

Comment:
Note: By adding this shebang and making the file executable, you can execute these files directly:

```shell
cd ~/src/edx/configuration/stanford/cme
./certificates.yml
```

instead of 

```shell
cd ~/src/edx/configuration/stanford/cme
ansible-playbook ./certificates.yml
```